trio_websocket/_impl.py

```diff
@@ -38,7 +38,6 @@
 CONN_TIMEOUT = 60 # default connect & disconnect timeout, in seconds
 MESSAGE_QUEUE_SIZE = 1
 MAX_MESSAGE_SIZE = 2 ** 20 # 1 MiB
-RECEIVE_BYTES = 4 * 2 ** 10 # 4 KiB
```

Comment:
Thank you for the attention here.  I have several concerns:
  1. 4k to 64k is a huge jump, and it could very well disrupt some existing use of the library in terms of performance, memory use, backpressure behavior, etc.   For example, we have a production deployment to 1000's of resource-constrained devices in the field, and this would be the kind of change that needs extensive testing (and my hunch is there would be some problems).
  2. if the new size doesn't work well, library users have no way to change it to something suitable
  3. the comment in trio clearly states that 64k was arbitrary, and starting from a small value and seeing how high you can go is prudent

I'd be interested in trying some modest increases (2x, 4x) on our specific deployment and trying to measure the effect, but I don't have the time available right now.  At best, if that looked promising, I'd consider making receive bytes able to be set in the API (including None), but still default to the current value.

Reply:
> At best, if that looked promising, I'd consider making receive bytes able to be set in the API (including None), but still default to the current value.

If you have a real app that is measurably benefitting from a larger buffer, that would be a useful datapoint.